src/templates/mht_part.tpl

```diff
@@ -0,0 +1,6 @@
+------=_NextPart_000_0000_01D0CFF2.897FE540
```

Comment:
So maybe we will use something more meaningful? Otherwise, we will soon forget that there is no 'magic' in this name.